cmd/metal-api/internal/ipam/ipam.go

```diff
@@ -154,14 +155,33 @@ func (i *ipam) PrefixUsage(ctx context.Context, cidr string) (*metal.NetworkUsag
 	if err != nil {
 		return nil, fmt.Errorf("prefix usage for cidr:%s not found %w", cidr, err)
 	}
-
+	pfx, err := netip.ParsePrefix(cidr)
+	if err != nil {
+		return nil, err
+	}
+	af := metal.IPv4AddressFamily
+	if pfx.Addr().Is6() {
+		af = metal.IPv6AddressFamily
+	}
+	availableIPs := metal.AddressFamilyUsage{
+		af: usage.Msg.AvailableIps,
+	}
+	usedIPs := metal.AddressFamilyUsage{
+		af: usage.Msg.AcquiredIps,
+	}
+	availablePrefixes := metal.AddressFamilyUsage{
+		af: usage.Msg.AvailableSmallestPrefixes,
+	}
+	usedPrefixes := metal.AddressFamilyUsage{
+		af: usage.Msg.AcquiredPrefixes,
+	}
```

Comment:
As this is always the same address family and the map will never have different ones it might be okay to just return the usage without address family and handle in the caller where to put it